packages/contracts/test/voting/addresslist-voting.ts

```diff
@@ -509,54 +508,54 @@ describe('AddresslistVoting', function () {
-        await voting.connect(signers[4]).vote(id, VoteOption.Yes, false);
-        expect(await voting.canExecute(id)).to.equal(false);
+        await voteWithSigners(voting, id, signers, {
+          yes: [0, 1, 2, 3, 4], // 5 votes
+          no: [], // 0 votes
+          abstain: [], // 0 votes
+        });
 
-        // `tryEarlyExecution` is turned on but the vote is not decided yet
-        let tx = await voting
-          .connect(signers[5])
-          .vote(id, VoteOption.Yes, true);
+        expect(await voting.canExecute(id)).to.equal(false);
 
-        expect(await voting.worstCaseSupport(id)).to.be.gt(
-          votingSettings.supportThreshold
-        );
-        expect(await voting.participation(id)).to.be.gte(
-          votingSettings.minParticipation
-        );
+        expect((await voting.getProposal(id)).executed).to.equal(false);
         expect(await voting.canExecute(id)).to.equal(false);
 
-        expect(await findEvent(tx, DAO_EVENTS.EXECUTED)).to.be.undefined;
+        // `tryEarlyExecution` is turned on but the vote is not decided yet
+        await voting.connect(signers[5]).vote(id, VoteOption.Yes, true);
+        expect((await voting.getProposal(id)).executed).to.equal(false);
+        expect(await voting.canExecute(id)).to.equal(false);
 
         // `tryEarlyExecution` is turned off and the vote is decided
-        tx = await voting.connect(signers[6]).vote(id, VoteOption.Yes, false);
-        expect(await findEvent(tx, DAO_EVENTS.EXECUTED)).to.be.undefined;
+        await voting.connect(signers[6]).vote(id, VoteOption.Yes, false);
+        expect((await voting.getProposal(id)).executed).to.equal(false);
+        expect(await voting.canExecute(id)).to.equal(false);
 
         // `tryEarlyExecution` is turned on and the vote is decided
-        tx = await voting.connect(signers[7]).vote(id, VoteOption.Yes, true);
-        expect(await findEvent(tx, DAO_EVENTS.EXECUTED)).to.be.undefined;
+        await voting.connect(signers[7]).vote(id, VoteOption.Yes, true);
+        expect((await voting.getProposal(id)).executed).to.equal(false);
+        expect(await voting.canExecute(id)).to.equal(false);
       });
 
       it('reverts if vote is not decided yet', async () => {
         await advanceIntoVoteTime(startDate, endDate);
 
-        await expect(voting.execute(id)).to.be.revertedWith(
-          customError('ProposalExecutionForbidden', id)
-        );
+        await expect(voting.execute(id))
+          .to.be.revertedWithCustomError(voting, 'ProposalExecutionForbidden')
+          .withArgs(id);
       });
     });
 
     context('Early Execution Mode', async () => {
       beforeEach(async () => {
         votingSettings.votingMode = VotingMode.EarlyExecution;
 
-        await voting.initialize(dao.address, votingSettings, addresslist(10));
+        await voting.initialize(
+          dao.address,
+          votingSettings,
+          [0, 1, 2, 3, 4, 5, 6, 7, 8, 9].map(i => signers[i].address)
```

Comment:
using `slice(...)` with `map(...)` would probably be cleaner: `signers.slice(0, 9).map(s => s.address)`

Reply:
Fixed here 26e42329bc5434e96fd75c0bb61139616678b262